web/docs/project/env-vars.md

```diff
@@ -92,40 +93,44 @@ These files should not be committed to version control, and they are already ign
 ### 2. Using Shell
 If you set environment variables in the shell where you run your Wasp commands (e.g., `wasp start`), Wasp will recognize them.
 
-You can set environment variables in the `.profile` or a similar file, or by defining them at the start of a command:
-
-```shell
-SOME_VAR_NAME=SOMEVALUE wasp start
-```
+You can set environment variables in the `.profile` or a similar file, which will set them permanently, or you can set them temporarily by defining them at the start of a command (`SOME_VAR_NAME=SOMEVALUE wasp start`).
```

Comment:
```suggestion
One option is setting your environment variables in `.profile` or a similar file, which will set them permanently,.
Another option is setting your environment variables temporarily by defining them at the start of a command (e.g., `SOME_VAR_NAME=SOMEVALUE wasp start`).
```

Reply:
How is that better? Didn't you just go passive, while it is better to speak directly to the user?

Reply:
"One option is setting" isn't passive form. It's an active sentence with a different subject.

I believe the whole thing is better because:
- The two sentences are shorter and simpler than the original longer sentence.
- The two sentences follow the same structure, contrasting the differences.
- The subjects of the two sentences also emphasize the two possibilities ("one option is", "another option is"). It's always best to use the most relevant actor in the sentence as the subject. In this case, that's "the way of setting environment variables"

Btw, I'm not hung up on every word I suggest in these changes. I just add them as examples (maybe I shouldn't be using the suggestion box). You can pick and apply what you think makes sense.

The core idea I care about here is shorter and simpler sentences. The rest of the stuff is just how I'd approach the core idea, but is not set in stone.

P.S. whatever you choose to do with this, there's an extra "the" before ".profile" you should remove. It may have slipped unnoticed.

Reply:
This is what I find tricky with these docs discussion: you say it sounds better to you, but to me original sounds better. I can read it in one piece with clear contract, versus (to me) unnatural break you went for in order to emphasize it is two options. And again it goes against the recommendation to talk to them directly, which you use as a quite strong argument in different places, but now you drop that.

Seems to me like for most of the way docs are written one can go and make a counter arugment: either splitting something, or combining something, or swtiching to "you", or switching away form it, all for the argument of "it is simpler / reads better this way", while it is really quite subjective and probalby has 0 impact on the total mass of readers who are also all unique people.

I don't think we shouldn't comment on docs in PRs at all, we should, and there are things in writing guide we should be looking to enforce indeed, but I think there is a line here, in the sense that if a suggestion is not clearly an objective improvement, it is probably better not make one. Or while making it, say very clearly it is just an alternative idea and author can do whatever they like, so they can take a look at it, take what they lilke, and move on, wihtout having to discuss it.

Reply:
I agree with the core idea behind your comment: _Everyone can choose their style, and we should emphasize when which suggestions are subjective alternatives._

So I'll keep that in mind for the future.

I do have one thing to clear up.

This part here:

> And again it goes against the recommendation to talk to them directly, which you use as a quite strong argument in different places, but now you drop that.

Isn't right. 

To avoid future confusion, I'll elaborate on "addressing the user directly" (and we can change the style guide to communicate it more clearly).

_Addressing the user directly_ means _write as if you were speaking to someone_, for example:
- "You can" instead of "We can."
- "Do this" instead of "Let's do this."
- "You can" instead of "The user can."
- "You should add a query" instead of "A query should be added." 

_Addressing the user directly_ doesn't mean _You must write every sentence in the second person, and every subject must be "you" (the user)._
That would be very impractical. Imagine if you were talking to me with the constraint that every sentence must start with "You." It's perfectly normal to use subjects that aren't the person you're speaking to (and we do it all the time, as we should).

So, the change I suggested here doesn't at all contradict the recommendation of addressing the user directly.

Anyway, here's what the writing guide currently says:
> **Address the reader directly.** Instead of "We can implement a Query..." or "A Query can be implemented..." or "The user can implement a Query...", say "You can implement a Query..." The docs should speak to the reader and address them directly to avoid possible confusion (e.g., Who are we? The Wasp team? The reader together with the Wasp team? etc.). Use the word user only to refer to the user of the software that your reader is developing. You can sometimes use first-person plural pronouns to refer to the organization (Wasp). For example, "We support both TypeScript and JavaScript" is OK, but "Wasp supports both TypeScript and JavaScript" is usually better.

How do you think we can improve it to make it less confusing?

Reply:
I get the idea, but based on this, to me it sounds like we should do "You can do thing X" instead of "One option is to do thing X". Shorter, speaks direct to the user. As writing guide suggests as the best way to phrase stuff. Would you make it longer + loose the direct talking to the user?